QWeb/internal/secrets.py

```diff
@@ -130,6 +166,37 @@ def _filtered_end_keyword(keyword: Keyword) -> None:
             LOGGER._other_loggers[0].log_message = debugfile_log
 
 
+def _filtered_end_library_keyword(data: Keyword, implementation: Keyword, result: Keyword) -> None:
+    """Modify Robot FW internal function "end_keyword".
+
+    This function removes secret data from "end keyword"
+    logs and returns previous log level.
+    """
```

Comment:
Specify that this function is for RFW version 7+

Reply:
done